BAZProjectC1/Model/Movie.swift

```diff
@@ -0,0 +1,11 @@
+//  Movie.swift
+//  BAZProjectC1
+
+import Foundation
+
+struct Movie: Codable {
```

Comment:
Esta struct no debería llamarse `Movie`. La otra struct es la que tiene los datos de la película.

Reply:
Pero el modelo dice que Result tiene el Arreglo de las Peliculas D: 

 "page": 1,
    "results": [
        {
            "adult": false,
            "backdrop_path": "/jsoz1HlxczSuTx0mDl2h0lxy36l.jpg",
            "id": 616037,
            "title": "Thor: Love and Thunder",
            "original_language": "en",
            "original_title": "Thor: Love and Thunder",
            "overview": "After his retirement is interrupted by Gorr the God Butcher, a galactic killer who seeks the extinction of the gods, Thor Odinson enlists the help of King Valkyrie, Korg, and ex-girlfriend Jane Foster, who now wields Mjolnir as the Mighty Thor. Together they embark upon a harrowing cosmic adventure to uncover the mystery of the God Butcher’s vengeance and stop him before it’s too late.",
            "poster_path": "/pIkRyD18kl4FhoCNQuWxWu5cBLM.jpg",
            "media_type": "movie",
            "genre_ids": [
                14,
                28,
                35
            ],
            "popularity": 4297.051,
            "release_date": "2022-07-06",
            "video": false,
            "vote_average": 6.806,
            "vote_count": 3511
        },

Reply:
Acomode el modelo al response, para que el codable pase transparente

Reply:
A lo que me refiero es que la otra struct tiene que llamarse Movie, porque realmente es la que describe a la película. Tiene el título de la película, el id, la portada, la reseña, etc., todo está relacionado con la película. Yo simplemente invertiría el nombre de las estructuras Movie -> Results y Results -> Movie.

Justo esto es a lo que en POO se le llama abstracción, sí la estructura describe las caracteristicas de una película tiene sentido que se llame `Movie`. `Results` es un nombre que no describe para nada los datos que contiene la estructura.